OMEdit/OMEditGUI/Editors/BaseEditor.cpp

```diff
@@ -688,6 +691,37 @@ PlainTextEdit::PlainTextEdit(BaseEditor *pBaseEditor)
   connect(this, SIGNAL(customContextMenuRequested(QPoint)), mpBaseEditor, SLOT(showContextMenu(QPoint)));
 }
 
+void PlainTextEdit::insertCompleterKeywords(QStringList keywords)
+{
+  for (int i = 0; i < keywords.size(); ++i)
+  {
+    mpStandardItem =new QStandardItem(keywords[i]);
```

Comment:
You could do,
`QStandardItem *pStandardItem = new QStandardItem(keywords[i]);`
and get rid of `mpStandardItem`.